network-api/networkapi/wagtailpages/forms.py

```diff
@@ -3,13 +3,17 @@
 
 class BuyersGuideProductCategoryForm(WagtailAdminModelForm):
     def clean_name(self):
+        instance = self.instance
+        set = instance.__class__.objects
         name = self.cleaned_data["name"]
-        if (
-            self.instance.__class__.objects.filter(name__iexact=name)
-            .exclude(pk=self.instance.pk)
-            .exists()
-        ):
+        duplicate = set.filter(name__iexact=name).exclude(pk=instance.pk)
+
+        if instance.hasattr('locale'):
```

Comment:
I believe it should be

```suggestion
        if hasattr(instance, 'locale'):
```